packages/elements/src/select/__demo__/index.html

```diff
@@ -29,32 +29,49 @@
       import(`../../../lib/select/themes/${theme}/${variant}/index.js`);
     </script>
 
+    <style>
+      section {
+        display: flex;
+        flex-direction: column;
+      }
+
+      section label {
+        margin-bottom: 4px;
+      }
+    </style>
+
     <demo-block header="Grouping" tags="groups,selected">
-      <ef-select placeholder="Pick item..." class="short">
-        <ef-item role="presentation" type="header">Drinks</ef-item>
-        <ef-item role="option" value="1">Cola</ef-item>
-        <ef-item role="option" selected value="2">Lemonade</ef-item>
-        <ef-item role="option" value="3">Orange Juice</ef-item>
-        <ef-item role="option" value="4" disabled>Apple Juice</ef-item>
-        <ef-item role="option" value="5">Iced Tea</ef-item>
-        <ef-item role="option" value="6">Water</ef-item>
-        <ef-item role="presentation" type="header">Ice Cream</ef-item>
-        <ef-item role="option" value="7">Vanilla</ef-item>
-        <ef-item role="option" value="8">Chocolate</ef-item>
-        <ef-item role="option" value="9">Honey &amp; Walnut</ef-item>
-        <ef-item role="option" value="10">Pistachio</ef-item>
-        <ef-item role="option" value="11">Salted Caramel</ef-item>
-        <ef-item role="option" value="12">Mint Choc Chip</ef-item>
-        <ef-item role="option" value="13">Hazelnut</ef-item>
-        <ef-item role="option" value="14">Strawberry</ef-item>
-        <ef-item role="option" value="15">Raspberry</ef-item>
-      </ef-select>
+      <section>
+        <label for="desserts">Desserts:</label>
+        <ef-select id="desserts" placeholder="Pick your preferred desserts...">
```

Comment:
The placeholder is always hidden because of the selected option. I think the placeholder should be removed.

Reply:
Remove this one's placeholder and updated `currency` demo to show placeholder kub.